__tests__/integration/basic.test.js

```diff
@@ -0,0 +1,9 @@
+describe('Basic end to end Sinopia BIBFRAME Editor', () => {
+  beforeAll(async () => {
+    await page.goto('http://127.0.0.1:8000/');
+  });
+
+  it('displays "Bibframe Editor Workspace" text', async () => {
```

Comment:
So I'm getting an error that this text is not found. If you run the `node server-bfe.js`, you get the following display that is missing this text.

<img width="1187" alt="screen shot 2018-10-02 at 10 20 49 am" src="https://user-images.githubusercontent.com/71847/46364428-02f45980-c633-11e8-8f89-128c0f1c2912.png">

Reply:
ah - whoops.  Probably something I changed in index.html.  Will investigate now.

(but ... the TEST passed!)

Reply:
Mine looks ok.  Did you npm install and grunt first?

![image](https://user-images.githubusercontent.com/96775/46366919-24544600-c631-11e8-8f6c-0607f551bbc5.png)